qtawesome/iconic_font.py

```diff
@@ -178,45 +179,94 @@ def _paint_icon(self, iconic, painter, rect, mode, state, options):
         if animation is not None:
             animation.setup(self, painter, rect)
 
-        font = iconic.font(prefix, draw_size)
-        # Disable font hinting to mitigate tremulous spinning to some extent
-        # See spyder-ide/qtawesome#39
-        if animation is not None:
-            font.setHintingPreference(QFont.PreferNoHinting)
-        painter.setFont(font)
         if 'offset' in options:
             rect = QRect(rect)
             rect.translate(round(options['offset'][0] * rect.width()),
                            round(options['offset'][1] * rect.height()))
 
-        if 'vflip' in options and options['vflip'] == True:
-            x_center = rect.width() * 0.5
-            y_center = rect.height() * 0.5
-            painter.translate(x_center, y_center)
-            transfrom = QTransform()
-            transfrom.scale(1,-1)
-            painter.setTransform(transfrom, True)
-            painter.translate(-x_center, -y_center)
-
-        if 'hflip' in options and options['hflip'] == True:
-            x_center = rect.width() * 0.5
-            y_center = rect.height() * 0.5
-            painter.translate(x_center, y_center)
-            transfrom = QTransform()
-            transfrom.scale(-1, 1)
-            painter.setTransform(transfrom, True)
-            painter.translate(-x_center, -y_center)
-
+        x_center = rect.width() * 0.5
+        y_center = rect.height() * 0.5
+        transform = QTransform()
+        transform.translate(+x_center, +y_center)
+        if 'vflip' in options and options['vflip'] is True:
+            transform.scale(1,-1)
+        if 'hflip' in options and options['hflip'] is True:
+            transform.scale(-1, 1)
         if 'rotated' in options:
-            x_center = rect.width() * 0.5
-            y_center = rect.height() * 0.5
-            painter.translate(x_center, y_center)
-            painter.rotate(options['rotated'])
-            painter.translate(-x_center, -y_center)
+            transform.rotate(options['rotated'])
+        transform.translate(-x_center, -y_center)
+        painter.setTransform(transform, True)
 
         painter.setOpacity(options.get('opacity', 1.0))
 
-        painter.drawText(rect, int(Qt.AlignCenter | Qt.AlignVCenter), char)
+        draw = options.get('draw')
+        if draw not in ('text', 'path', 'glyphrun', 'image'):
+            # Use QPainterPath when setting an animation
+            # to fix tremulous spinning icons.
+            # See #39
```

Comment:
Let change the comment to follow other comments pointing to issues:
```suggestion
            # See spyder-ide/qtawesome#39
```